portfolio/src/main/webapp/second.html

```diff
@@ -0,0 +1,25 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>About me</title>
+    <link rel="stylesheet" href="style.css">
+    <div class="topnav">
+        <a class="active" href="index.html">Home</a>
+        <a href="#second">About me</a>
+        <a href="projects.html">Projects</a>
+        <a href="skills.html">Skills</a>
+        <a href="contact.html">Contact</a>
+    </div>
+</head>
+<header id="second">
+    <h1> About me </h1><br>
+    <section id="seaction1">
+        <div class="box2" <p> My name is Siham Abdi and I am a student at the University of Minnesota studying computer science and Sociology. I love learning and trying new things. During my free time I watch shows, read, or skateboard. I grew up in a city 1.5 hours from
+            Minneapolis, known as Rochester Minnesota. While I lived there I had only been introduced to the health care field, because it is the home of Mayo Clinic. Once I moved to Minneapolis, I found my passion for technology. </p>
+        </div>
+        <img src='../img/aboutme.jpeg' alt="HTML" width="400" height="350">
```

Comment:
The "alt" tag is for users who cannot view the content, so you want to make it descriptive to what the picture contains. Here, you might put something like "Picture of me" or something like that depending on what your image is.